sdk/azcore/error.go

```diff
@@ -7,47 +7,29 @@ package azcore
 
 import (
 	"errors"
-	"fmt"
+	"net/http"
+
+	azruntime "github.com/Azure/azure-sdk-for-go/sdk/internal/runtime"
```

Comment:
Just wondering if the alias should be azruntime? Doesn't really matter but it implies azure runtime, not necessarily bad, but possibly misleading. Anyways, not going to block on that, just some thoughts about it.

Reply:
I did this to disambiguate between our internal `runtime` package and the one in the standard library.  I picked `az` as we seem to prefix all of our hand-authored track 2 content with it.

Reply:
Yes I figured, just thought if we ever have an azruntime package in the future it could be weird. Not that it would ever be imported here. I thought maybe 'sdkruntime' or 'internalruntime' to differentiate internal packages, but I dont think either of those are good aliases either. So we can leave it like this.

Reply:
I like `sdkruntime`, I'll switch to that.

Reply:
Ok sounds good to me 😊